src/openforms/registrations/contrib/microsoft_graph/plugin.py

```diff
@@ -1,29 +1,55 @@
+from pathlib import PurePosixPath
 from typing import NoReturn
 
 from django.urls import reverse
+from django.utils import timezone
 from django.utils.text import slugify
 from django.utils.translation import gettext_lazy as _
 
-from openforms.contrib.microsoft.client import MSGraphClient, MSGraphUploadHelper
+from openforms.contrib.microsoft.client import (
+    MSGraphClient,
+    MSGraphOptions,
+    MSGraphUploadHelper,
+)
 from openforms.contrib.microsoft.exceptions import MSAuthenticationError
 from openforms.plugins.exceptions import InvalidPluginConfiguration
 from openforms.registrations.contrib.microsoft_graph.models import (
     MSGraphRegistrationConfig,
 )
 from openforms.submissions.models import Submission, SubmissionReport
 from openforms.submissions.tasks.registration import set_submission_reference
+from openforms.template import render_from_string
 
 from ...base import BasePlugin
 from ...exceptions import NoSubmissionReference, RegistrationFailed
 from ...registry import register
+from .config import MicrosoftGraphOptionsSerializer
 
 
 @register("microsoft-graph")
 class MSGraphRegistration(BasePlugin):
     verbose_name = _("Microsoft Graph (OneDrive/SharePoint)")
+    configuration_options = MicrosoftGraphOptionsSerializer
+
+    def _get_folder_name(
+        self, submission: Submission, options: MSGraphOptions
+    ) -> "PurePosixPath":
+        now_utc = timezone.now()
+        date = timezone.localtime(now_utc).date()
```

Comment:
@SilviaAmAm I've tweaked this one because `timezone.now().date()` would return the day before if you run it between 00:00 and 01:00 at night due to timezone issues.

The correct approach is grabbing `now` in UTC, converting that to the localtime (according to `settings.TIMEZONE`) and _then_ grabbing the date portion.